customer-deployment/serverless.yml

```diff
@@ -11,9 +11,12 @@ provider:
   ignore_certs: true
 
 custom:
-  customer-namespace: ${opt:customer-namespace, file(package.json):customer-namespace}
-  customer-package: ${opt:customer-package, file(package.json):customer-package}
-  bindings-namespace: ${opt:bindings-namespace, file(package.json):bindings-namespace}
+  # Adobe I/O Runtime configuration. The variables can be overriden by command-line arguments.
+  customer-namespace: ${opt:customer-namespace, 'cjelger'}
```

Comment:
I think we should move these to a separate config file as well or use placeholder here. But not store personal namespace info here.

Reply:
Sure my mistake, this should be replaced by a placeholder. BTW, *you* asked me to remove that from `package.json` so I don't think we should ask an extra config file, but we can also ad those in `credentials.json`.

Reply:
`credentials.json` is fine